.eslintrc.js

```diff
@@ -8,3 +8,5 @@ module.exports = {
   },
   plugins: ['jest']
 };
+
```

Comment:
There are multiple empty lines at the end of the file. Consider deleting these to maintain a clean and readable codebase.